Doc/library/collections.rst

```diff
@@ -1252,16 +1252,17 @@ variants of :func:`functools.lru_cache`:
                 self.cache.move_to_end(args)
                 return self.cache[args]
             result = self.func(*args)
-            self.requests[args] = self.requests.get(args, 0) + 1
+            self.requests[args] = self.requests.get(args, 0) + 1 # increment existing value by 1 if seen before, otherwise set to 1
             if self.requests[args] <= self.cache_after:
                 self.requests.move_to_end(args)
                 if len(self.requests) > self.maxrequests:
-                    self.requests.popitem(0)
+                    self.requests.popitem(last=False)
             else:
-                self.requests.pop(args, None)
+                # entry has been seen more than cache_after times
```

Comment:
This is comment is clear; however, I'm dubious that it adds value because the code already reads as "if requests[args] < cache_after: ... else: ..."

There isn't much of a point in having a comment that says exactly what the code already says.

Reply:
I added this comment using the same reasoning I mentioned above, and agree this comment is also repetitive.

I also updated this part of the example in commit ecbb79a2ae723a7e8baf57bce1f3a05feaaab4db.